File: day7-array/demo10.js

   
   
function demo(){
    var arr=["123","55","253","789"];
    var arr2=["45","78","111","55","253","45"];
    arr=arr.concat(arr2);
    for(var i=0;i<arr.length;i++){
        for(var j=i+1;j<arr.length;j++){
            if(arr[i]==arr[j]){
                arr.splice(j,1);
                j--;
            }
        }
    }
    arr.sort(function(a,b){
        return a-b;
    });
    console.log(arr.join("<")); // 45<55<78<111<123<253<789
}

File: day7-array/demo10.java

import java.util.Arrays;
import java.util.List;
import java.util.stream.Collectors;
import java.util.stream.Stream;

public class demo10 {
	public static void main(String[] args) {
		String[] arr = {"123","55","253","789"};
		String[] arr2 = {"45","78","111","55","253","45"};
		List<String> list1 =Arrays.asList(arr);
		List<String> list2 =Arrays.asList(arr2);
		System.out.println(Stream.concat(list1.stream(), list2.stream()).distinct()
				.sorted((a,b)->Integer.parseInt(a)-Integer.parseInt(b))
				.collect(Collectors.joining("<"))); 
		// 45<55<78<111<123<253<789
	}
}